StandAlone/share/qml/AboutDialog.qml

```diff
@@ -54,13 +54,13 @@ Dialog {
             }
             TabButton {
                 //: Program license.
-                text: qsTr("License")
+                text: qsTr("Licence")
             }
             TabButton {
                 /*: License for 3rd party components used in Webcamoid, like
                     libraries and code snippets.
                  */
-                text: qsTr("3rd Party Licenses")
+                text: qsTr("3rd Party Licences")
```

Comment:
And, this one.

Reply:
Something weird going on with github! My local branch (which I force pushed to my github) does not contain those "License" changes - those were all contained in a 3rd commit which I dropped from the branch. Force-pushing once more now!
```
$ git status -u no                                                                                                                                                                              
On branch pr-aboutdialog-improvements                                                                                                                                                                                                       
nothing to commit, working tree clean                                                                                                                                                                                                       
tj@sunny:/srv/NAS/Sunny/SourceCode/webcamoid$ gitlog -n 5  
21f0812c 2022-06-01 11:04:20 +0100 N Tj Merge branch 'feature-aboutdialog-size-scaling' into pr-aboutdialog-improvements                                                                                                                    
51c253d2 2022-05-31 16:57:27 +0100 N Tj Merge branch 'fix-about-thirdpartylicences-truncated-text' into pr-aboutdialog-improvements                                                                                                         
4f2da062 2022-05-31 16:50:32 +0100 N Tj AboutDialog: allow dialog to scale with application window                    
7538b07f 2022-05-31 16:27:01 +0100 N Tj ThirdPartyLicences: make truncated text wrap                                  
f25b8b1c 2022-05-19 07:19:12 +0200 E AHOHNMYC Translated using Weblate (Russian)
$ git diff f25b8b1c                                                       
diff --git a/StandAlone/share/qml/AboutDialog.qml b/StandAlone/share/qml/AboutDialog.qml                              
index 2701f2e5..65576838 100644                                                                                       
--- a/StandAlone/share/qml/AboutDialog.qml                                                                                                                                                                                                  
+++ b/StandAlone/share/qml/AboutDialog.qml                                                                            
@@ -26,8 +26,8 @@ import Webcamoid 1.0                                                                                
 Dialog {                                         
     id: aboutDialog                                                                                                  
     standardButtons: Dialog.Close                                                                                    
-    width: AkUnit.create(640 * AkTheme.controlScale, "dp").pixels                                                    
-    height: AkUnit.create(400 * AkTheme.controlScale, "dp").pixels
+    width: wdgMainWidget.width * 0.75                     
+    height: wdgMainWidget.height * 0.75                                                                              
     modal: true                                                                                                      
     title: qsTr("About %1").arg(mediaTools.applicationName)                                                          
                                                                                                                      
diff --git a/StandAlone/share/qml/ThirdPartyLicenses.qml b/StandAlone/share/qml/ThirdPartyLicenses.qml                
index b691ac76..ee17ca1e 100644                                                                                       
--- a/StandAlone/share/qml/ThirdPartyLicenses.qml                                                                                                                                                                                           
+++ b/StandAlone/share/qml/ThirdPartyLicenses.qml                                                                                                                                                                                           
@@ -37,11 +37,13 @@ Page {                                                                                            
                 text: "The followings are the Licenses for 3rd-party work incorporated into Webcamoid. <b>These Licenses DOES NOT applies to Webcamoid itself.</b>"                                                                        
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                                                                               
+                Layout.maximumWidth: parent.width                                                                                                                                                                                          
             }                                                                                                        
             Label {                                                                                                  
                 text: "License for resources taken from openclipart.org:"                                            
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                                                                               
+                Layout.maximumWidth: parent.width                                                                    
             }                                                                                                        
             TextArea {                                    
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/openclipart.txt")                    
@@ -52,6 +54,7 @@ Page {                
                 text: "License for code taken from OpenCV:"                                                          
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                    
+                Layout.maximumWidth: parent.width                                                                    
             }                                                                                                        
             TextArea {                                                                                               
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/OpenCV.txt")                         
@@ -62,6 +65,7 @@ Page {              
                 text: "License for code and algorithms used in Temperature plugin:"                                  
                 wrapMode: Text.WordWrap                                                                                                                                                                                                    
                 Layout.fillWidth: true                                                                                                                                                                                                     
+                Layout.maximumWidth: parent.width                                                                    
             }                                                                                                        
             TextArea {                                                                                               
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/TemperatureAlgorithm.txt")           
@@ -72,6 +76,7 @@ Page {                                                                                              
                 text: "License for the usb.ids file:"                                                                
                 wrapMode: Text.WordWrap                                                                              
                 Layout.fillWidth: true                                                                               
+                Layout.maximumWidth: parent.width                                                                    
             }                          
             TextArea {                
                 text: mediaTools.readFile(":/Webcamoid/share/3rd-party/licenses/UsbIds.txt")
```